doc/installation/system/inifile.rst

```diff
@@ -52,6 +54,14 @@ The file should contain the following contents::
 You may want to use the MySQL database or Maria DB. There are two possible
 drivers, to connect to this database. Please read :ref:`mysqldb`.
 
+``SQLALCHEMY_ENGINE_OPTIONS`` is a dictionary of keyword args to send
+to `create_engine() <https://docs.sqlalchemy.org/en/14/core/engines.html#sqlalchemy
+.create_engine>`_. `max_identifier_length <https://docs.sqlalchemy.org/en/14/core/engines
+.html#sqlalchemy.create_engine.params.max_identifier_length>`_ is the database’s
+configured maximum number of characters that may be used in a SQL identifier
+such as a table name, column name, or label name. ``max_identifier_length``
+should be set to *128* to avoid error messages.
```

Comment:
Please replace

    ``max_identifier_length`` should be set to *128* to avoid error messages.

with

      For Oracle version 19 and above the `max_identifier_length` should be set to 128.

Since setting this value is only necessary for Oracle installations.

Reply:
Fixed in 311611ae32b43837bfd4427d1cd6766ab188361c.